mmpretrain/datasets/gqa_dataset.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+from typing import List
+
+import mmengine
+from mmengine.dataset import BaseDataset
+
+from mmpretrain.registry import DATASETS
+
+
+@DATASETS.register_module()
+class GQA(BaseDataset):
+    """GQA dataset.
+
```

Comment:
Consider note the dataset class use LAVIS format and provide the download link of the annotation files.
And how to organize the dataset folder.